git-ftp

```diff
@@ -1167,6 +1168,15 @@ set_curl_proxy() {
 	[ -z "$CURL_PROXY" ] && CURL_PROXY="$(git config --get http.proxy)"
 }
 
+set_merge_args() {
+	local config="$(get_config no-commit)"
+	[ -n "$config" ] && NO_COMMIT=1
+	
+	if [ $NO_COMMIT -eq 1 ]; then
+		MERGE_ARGS="$MERGE_ARGS --no-commit --no-ff"
```

Comment:
Another good improvement would be to turn this string into an array to allow proper quoting.

Reply:
Yes. I also thought about it for a moment, but then left it as it was, because [insert poor excuse here 😉]